File: SmplID.Derayah.VB/Models/HCM/Role.vb
Namespace Models.HCM
    Public Class Role
        Public Property id As String
        Public Property name As String
        Public Property displayName As String
        Public Property description As String
        Public Property members As List(Of HCMRoleMember)
    End Class

    Public Class HCMRoleMember
        Public Property value As String
        Public Property operation As String
    End Class
    Public Class UsersInRoles
        Public Property userId As String
        Public Property roleId As String
    End Class
End Namespace

File: SmplID.Derayah/Models/HCM/Role.cs
using System.Collections.Generic;

namespace SmplID.Derayah.Models.HCM
{
    public class Role
    {
        public string id { get; set; }
        public string name { get; set; }
        public string displayName { get; set; }
        public string description { get; set; }
        public List<HCMRoleMember> members { get; set; }
    }
    public class HCMRoleMember
    {
        public string value { get; set; }
    }
}
